src/sentry/workflow_engine/processors/data_condition_group.py

```diff
@@ -86,17 +85,28 @@ def evaluate_data_conditions(
 
 def process_data_condition_group(
     data_condition_group_id: int,
-    value: Any,
+    value: T,
```

Comment:
I'm not totally sure if the typevar is helpful here since we're not associating it with a return value or tying it to other types

Reply:
Yeah, I was just thinking it's a bit more illustrative of its intended use - since everything else is tied to a generic.

I was also hoping that at some point I might see some way to tie them together 🤣